components/infobox/wikis/apexlegends/infobox_league_custom.lua

```diff
@@ -67,19 +60,14 @@ function CustomInjector:parse(id, widgets)
 			Cell{name = 'Platform', content = {self.caller:_getPlatform()}}
 		)
 	elseif id == 'liquipediatier' then
-		local algsTier = args.algstier
-		if String.isNotEmpty(algsTier) then
+		local publisherTier = args.publishertier or args.algstier
+		if String.isNotEmpty(publisherTier) then
 			table.insert(widgets, 1, Cell{
 				name = 'ALGS circuit tier',
-				content = {'[[Apex Legends Global Series|' .. algsTier .. ']]'},
+				content = {'[[Apex Legends Global Series|' .. publisherTier .. ']]'},
 				classes = {'tournament-highlighted-bg'}
 			})
 		end
-		table.insert(widgets, Cell{
-			name = 'EA tier',
-			content = {EA_TIERS[string.lower(args.eatier or '')]},
-			classes = {'tournament-highlighted-bg'}
-		})
```

Comment:
This breaks pages that have an EA tier but aren't part of ALGS and/or have different EA and ALGS tiers, no?

Reply:
nvm, haven't seen the comment stating eatier is unused